src/Microsoft.IIS.WebManager/app/main/settings.ts

```diff
@@ -0,0 +1,65 @@
+export var SETTINGS : { [key:string]: string} = {
+    "version": "2.2.2",
+    "api_version": "2.2.0",
+    "api_setup_version": "2.2.0",
+    "api_download_url": "http://go.microsoft.com/fwlink/?LinkId=829373",
+    "ga_track": "UA-XXXXXXXX-X",
+}
+
+export const WebSiteListComponentName = "WebSiteListComponent"
+export const AppPoolComponentName = "AppPoolListComponent"
+export const WebFilesComponentName = "WebFilesComponent"
+export const WebAppListComponentName = "WebAppListComponent"
+export const VdirListComponentName = "VdirListComponent"
+export const AuthenticationComponentName = "AuthenticationComponent"
+export const AuthorizationComponentName = "AuthorizationComponent"
+export const CertificatesComponentName = "CertificatesComponent"
+export const CentralCertificateComponentName = "CentralCertificateComponent"
+export const DefaultDocumentsComponentName = "DefaultDocumentsComponent"
+export const DirectoryBrowsingComponentName = "DirectoryBrowsingComponent"
+export const IpRestrictionsComponentName = "IpRestrictionsComponent"
+export const LoggingComponentName = "LoggingComponent"
+export const MimeMapsComponentName = "MimeMapsComponent"
+export const MonitoringComponentName = "MonitoringComponent"
+export const ModulesComponentName = "ModulesComponent"
+export const CompressionComponentName = "CompressionComponent"
+export const RequestFilteringComponentName = "RequestFilteringComponent"
+export const HttpResponseHeadersComponentName = "HttpResponseHeadersComponent"
+export const RequestTracingComponentName = "RequestTracingComponent"
+export const StaticContentComponentName = "StaticContentComponent"
+export const UrlRewriteComponentName = "UrlRewriteComponent"
+export const FilesComponentName = "FilesComponent"
+export const UploadComponentName = "UploadComponent"
+export const WarningComponentName = "WarningComponent"
+export const AppModuleName = "AppModule"
+
+export class ComponentReference {
+    constructor(public name: string, public ico: string, public component_name: string, public api_name: string, public api_path: string) {}
+}
+
+export const FilesComponentReference: ComponentReference = new ComponentReference("Files", "fa fa-files-o", FilesComponentName, "files", "/api/webserver/url-rewrite/{id}")

```

Comment:
Files component points to url rewrite?

Reply:
What is this used for? There is a files componentreference in the GLOBAL_MODULES.